diff --git a/bin/cuirass.in b/bin/cuirass.in
index 207eecf..5514466 100644
--- a/bin/cuirass.in
+++ b/bin/cuirass.in
@@ -3,6 +3,7 @@
 # @configure_input@
 #GUILE_LOAD_PATH="@PACKAGE_LOAD_PATH@${GUILE_LOAD_PATH:+:}$GUILE_LOAD_PATH"
 #GUILE_LOAD_COMPILED_PATH="@PACKAGE_LOAD_COMPILED_PATH@${GUILE_LOAD_COMPILED_PATH:+:}$GUILE_LOAD_COMPILED_PATH"
+export GUILE_JIT_THRESHOLD=-1
 exec ${GUILE:-@GUILE@} --no-auto-compile -e main -s "$0" "$@"
 !#
 ;;;; cuirass -- continuous integration tool
